Casks/m/macpar-deluxe.rb

```diff
@@ -8,12 +8,24 @@
   homepage "https://gp.home.xs4all.nl/Site/MacPAR_deLuxe.html"
 
   livecheck do
-    url :homepage
-    regex(/Release\s+notes\s+for\s+version\s+(\d+(?:\.\d+)+)/i)
+    url "https://gp.home.xs4all.nl/versionlist.xml"
+    strategy :xml do |xml|
+      xml.elements["//string"]&.text&.strip
```

Comment:
```suggestion
      version = xml.elements["//key[text()='MacPARdeLuxe']"]&.next_element&.text
      next if version.blank?

      version.strip
```

This ensures that the `livecheck` will continue to work if the version is not the first `<string>` element in the XML document.

Reply:
Yeah, fair point.